src/components/UserCard.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="student" :class="themeClass">
+    <div class="student__name">{{name}}</div>
+    <img class="student__avatar" :src="avatar" alt />
+    <router-link class="student__user" :to="'/about/'+user">{{user}}</router-link>
```

Comment:
As a suggestion, you can use template literals in `:to` value, in order to be more readable